lib/gitx/cli/update_command.rb

```diff
@@ -20,11 +20,9 @@ def update
       private
 
       def update_branch(branch)
-        begin
-          run_cmd "git pull origin #{branch}"
-        rescue
-          fail MergeError, 'Merge Conflict Occurred. Please fix merge conflict and rerun the update command'
-        end
+        run_cmd "git pull origin #{branch}"
+      rescue
+        raise MergeError, 'Merge Conflict Occurred. Please fix merge conflict and rerun the update command'
```

Comment:
Line is too long. [107/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.